mitiq/benchmarks/mirror_circuits.py

```diff
@@ -149,7 +149,7 @@ def generate_mirror_circuit(
             ``cirq.Circuit``.
 
     Returns:
-        A randomized mirror circuit.
+        A randomized mirror circuit and correct bitstring.
```

Comment:
We can say `A tuple of a randomized mirror circuit of type return_type, and the correct bitstring`.
I left a comment in #2344: what's the meaning of correct here?